MaterialLib/MPL/Properties/CapillaryPressureSaturation/CapillaryPressureRegularizedVanGenuchten.cpp

```diff
@@ -0,0 +1,130 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on April 20, 2020, 9:30 AM
+ */
+
+#include "CapillaryPressureRegularizedVanGenuchten.h"
+
+#include <algorithm>
+#include <cmath>
+
+#include "BaseLib/Error.h"
+#include "MaterialLib/MPL/Medium.h"
+#include "MaterialLib/MPL/VariableType.h"
+#include "MaterialLib/MPL/Utils/CheckVanGenuchtenExponentRange.h"
+
+namespace MaterialPropertyLib
+{
+void checkSaturationRange(const double Sl)
+{
+    if (Sl < 0 || Sl > 1)
+    {
+        OGS_FATAL("The saturation of {:g} is  out of its range of [0, 1]", Sl);
+    }
+}
+
+CapillaryPressureRegularizedVanGenuchten::
+    CapillaryPressureRegularizedVanGenuchten(
+        double const residual_liquid_saturation,
+        double const maximum_liquid_saturation,
+        double const exponent,
+        double const p_b)
+    : Sl_r_(residual_liquid_saturation),
```

Comment:
No need to store the value if it is only used in the ctor.

Reply:
Removed.